### DISCUSSION TITLE
PHP8.2 for Ubuntu 18.04 (Bionic)

### DISCUSSION THREAD

**Discussion by petegould2023** [oerdnj/deb.sury.org#1979] [General]
Hi. Is PHP8.2 available for Ubuntu in the ppa repository? I did 'add-apt-repository ppa:ondrej/php' and 'apt-get install php8.2', but it couldn't find it. If I manually change the source list to Focal is will find PHP8.2, but there are dependency issues. I'm trying to get Ubuntu 18.04 onto a supported version of PHP, but the only version I can install is PHP7.2. Any help would be much appreciated. Pete

**Comment by stpnkcrk** [comment]
#1922

**Reply by petegould2023** [reply]
Many thanks for replying - I thought I was going crazy when the instructions I found didn't work! Time to upgrade my Ubuntu release I think. Pete
